docs/translation-guide.md

```diff
@@ -14,6 +14,8 @@ Once you’ve made a selection, you can provide suggestions for strings that hav
 
 When translating you may encounter a situation where you have to decide between several ways of saying the same thing. In these situations we refer to the Ubuntu [general translator guide](https://help.launchpad.net/Translations/Guide), and for language specific issues we follow Ubuntu's [team translation guidelines](https://translations.launchpad.net/+groups/ubuntu-translators). Following these guides ensure uniform translations, while also allowing anyone to contribute.
 
+If you however don't want to use Rosetta or want to make a lot a changes at ones, you can also translate offline. To do so you must firsty go to a project you want to translate, choose your language and then click on "Download translation". You are now able to request an e-mail, which gives you the right link to download the translation. Afterwards choose your favourite text- or po-editor (e.g. [Poedit] (https://apps.ubuntu.com/cat/applications/raring/poedit/)) to start translating. To then upload it back to Rosetta just use "Upload translation" on the same page as where you downloaded it.
```

Comment:
"changes at ones" -> "changes at once"
"firsty" -> "firstly"
"favourite" -> "favorite" (we use US spellings)

In general I think it's better to break steps up into a list than to do them in a paragraph form.